### DISCUSSION TITLE
Error in .riv file written by Flopy 3.3.5

### DISCUSSION THREAD

**Discussion by Hydrogeologist** [modflowpy/flopy#1722] [Q&A] [answered]
I have the following error in .riv file written by Flopy (a dot is placed between stage and conductance when conductance > 10E+6):

RIV package for MODFLOW-2005 generated by Flopy 3.3.5
   472        50
   472         0 # stress period 1
     1       223        61    410.89    150625    405.89
     1       224        60    407.541.15938E+06    402.54
     1       224        61    409.34    991250    404.34
     1       225        59    401.94    890625    396.94
     1       225        60    404.341.18375E+06    399.34
     1       226        58    397.23    853750    392.23
     1       226        59    399.331.38125E+06    394.33

I would be glad to know if I am missing something there.

Thanks in advance!

**Comment by jlarsen-usgs** [comment, marked as answer]
@Hydrogeologist 

Just out of curiousity, do you have the free format flag enabled in your bas package/for your model? My first thought is that it looks like the value's precision is completely filling a fixed width column. If the free format option is not enabled the width of floating points is 10 digits. To check and change from fixed format files to free format:

```python
print(model.free_format_input)

# change to free format
model.free_format_input = True  

# change to fixed format
model.free_format_input = False

model.write_input()
```

**Reply by Hydrogeologist** [reply]
Many thanks for the quick response! The free format option was not enabled. It worked after changing from fixed to free format.
